### DISCUSSION TITLE
Fooocus does not launch 😢

### DISCUSSION THREAD

**Discussion by ElodiEdina** [lllyasviel/Fooocus#2620] [Q&A]
Hi everybody

I use Stability Matrix to launch Fooocus and Stable Diffusion, but since a few days now, even if on the Stability Matrix interface everything seems ok, Fooocus does not open a new tab to be displayed on, then I cannot use it 😢😢😢😢😢

Where come from this issue and how can I fix it, please ?

Thank you very much in advance for your help

**Comment by ILikeToasters** [comment]
Without seeing what is actually happening there is no way to know.  Would need to see the console log. Most likely it says something there that will give you an idea why.

**Reply by ElodiEdina** [reply]
No, I mean yes..; I mean I saw the Stability Matrix console but regarding to all what has been done everything's alright.... I will make a screencap and it here... I come back in some minutes

**Reply by ElodiEdina** [reply]
Here I am with the screencap 
![smp](https://github.com/lllyasviel/Fooocus/assets/156558683/17170bc3-6d1d-45ff-8b2e-5a1f02685a02)

**Reply by ILikeToasters** [reply]
Not sure why it is not opening automatically in the browser but have you clicked on the link? "`http://127.0.0.1:7865/`"

**Reply by ElodiEdina** [reply]
Usually it opens automatically, and recently I used to click on the button, but nothing happened 😟 I can try again, if you want, I am really sure that won't do anything (except freezing all my browser and every app on my computer, of course 😂)

**Reply by ElodiEdina** [reply]
PS : the link "http://127.0.0.1:7865/" is not clickable

**Reply by ILikeToasters** [reply]
If you click on the link it should open in browser. Or even copy and paste that address into your browser window address bar. It says it is running.

**Reply by ElodiEdina** [reply]
It doesn't, and I cannot copy-past, or maybe it is too slow, then I can, but I didn't see it

**Reply by ILikeToasters** [reply]
I am not really that familiar with Stability Matrix. But that should be the address.

**Reply by ILikeToasters** [reply]
Just type it in the address window. That will tell you for sure if running. It says it is.

**Reply by ElodiEdina** [reply]
But what surprises me the most is that I use to use Fooocus with not any issue not so long ago

**Reply by ElodiEdina** [reply]
Okay, I'll do it !!!! Thank you very much, I come back to tell you 🥰🥰🥰🤗🤗🤗

**Reply by ILikeToasters** [reply]
I just tested myself with a fresh install of Stability Matrix and that should all be clickable. Sounds like it might be an issue with Stability Matrix and not Fooocus.

**Reply by ElodiEdina** [reply]
😭😭😭😭😭😭😭 Oh nooo ! Not that !!! I need my Stability Matrix !!!!! And there not any option to fix it without uninstall/re-install, like Pytorch

**Comment by ElodiEdina** [comment]
When I copy the link, I succeed in reaching the UI of Fooocus !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! 😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍🤩🤩🤩🤩🤩🤩🤩🤩🤩🤩🤩🤩🤩🤩🤩🤩🤩🤩🤩🤩🤩🤩🤩🤩🤩🤩🤩🤩🤩🤩🤩🤩🤩🤩🤩🤩🤩🤩🤩🤩🤩🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳

**Reply by ElodiEdina** [reply]
Something changed... Before, it was automatically launched, no need to copy anything to make it work 🤔🤔🤔🤔🤔🤔

**Reply by ElodiEdina** [reply]
Anyway, THANK YOU SOOOOOOOOOOOOOOOOOOOOOOOOOOOO MUCH for the help !!!!!!!!!!!!!! You are FANTASTIC !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! 🥰🥰🥰🥰🥰🥰🥰🥰🥰🥰🥰🥰🥰🥰🥰🤩🤩🤩🤩🤩🤩🤩🤩🤩🤩🤩🤩🤩🤩🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🤗🤗🤗🤗🤗🤗🤗🤗🤗🤗✨✨✨✨✨✨✨✨✨✨✨✨🌈🌈🌈🌈🌈🌈🌈🌈🌈🌈🌈🌈🌈☀️☀️☀️☀️☀️☀️☀️☀️☀️☀️☀️☀️🎊🎊🎊🎊🎊🎊🎊🎊🎊🎊🎊🎊🎊🎊🎊🎊🎊🎊🎊🎊🎊🎊🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉

**Reply by ILikeToasters** [reply]
No problem. Like to keep people enjoying Fooocus.

**Reply by ElodiEdina** [reply]
Thank you !!!!!!!!!!!!! You're the Best !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! 🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗🥰🤗

**Reply by ElodiEdina** [reply]
Well, it actually refuses to generate now 😭😭😭😭😭😭😭
